raiden/transfermanager.py

```diff
@@ -86,68 +128,172 @@ def transfer_async(self, amount, target, identifier=None, callback=None):
         """
 
         # Create a default identifier value
-        if not identifier:
+        if identifier is None:
```

Comment:
`assert isinstance(identifier, (long, int, None))` at the start (thus becoming part of the documentation) of the method.